docs/source/en/api/pipelines/stable_diffusion/adapter.mdx

```diff
@@ -0,0 +1,189 @@
+
+## Available Pipelines:
+
+| Pipeline | Tasks | Demo
+|---|---|:---:|
+| [StableDiffusionAdapterPipeline](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_adapter.py) | *Text-to-Image Generation with T2I-Adapter Conditioning* | -
+
+## Usage example
+
+In the following we give a simple example of how to use a *T2IAdapter* checkpoint with Diffusers for inference.
+The inference pipeline is the same for all pipelines:
+
+ 1. Take an image and run it through a pre-conditioning processor to obtain *control image*.
+ 2. Run the pre-processed *control image* and *prompt* through the [`StableDiffusionAdapterPipeline`].
+
+Let's have a look at a simple example using the [Color Adapter](https://huggingface.co/RzZ/sd-v1-4-adapter-color).
+
+```python
+from diffusers.utils import load_image
+
+image = load_image("https://huggingface.co/RzZ/sd-v1-4-adapter-color/resolve/main/color_ref.png")
+```
+
+![img](https://huggingface.co/RzZ/sd-v1-4-adapter-color/resolve/main/color_ref.png)
+
+
+Then we can create our color palette by simply resize it to 8 by 8 pixels then scale it back to original size.
+
+```python
+from PIL import Image
+
+color_palette = image.resize((8, 8))
+color_palette = color_palette.resize((512, 512), resample=Image.Resampling.NEAREST)
+```
+
+Let's take a look at the processed image.
+
+![img](https://huggingface.co/RzZ/sd-v1-4-adapter-color/resolve/main/color_palette.png)
+
+
+After we having `color_palette` in hand, we can create the [`StableDiffusionAdapterPipeline`] with pretrained checkpoint.
+
+```py
+import torch
+from diffusers import StableDiffusionAdapterPipeline, T2IAdapter
+
+adapter = T2IAdapter.from_pretrained("RzZ/sd-v1-4-adapter-color")
+pipe = StableDiffusionAdapterPipeline.from_pretrained(
+    "CompVis/stable-diffusion-v1-4",
+    adapter=adapter,
+    torch_dtype=torch.float16,
+)
+pipe.to("cuda")
+```
+
+And finally we feed the data to the pipelien and wait for the result!
+
+```py
+# fix the random seed, so you will get the same result as the example
+generator = torch.manual_seed(7)
+
+out_image = pipe(
+    ["At night, glowing cubes in front of the beach"],
+    image=[color_palette],
+    generator=generator,
+).images[0]
```

Comment:
Do we always have to pack these inputs as lists? For single inputs, I would imagine having them as lists. But okay for me.

Reply:
removed!